tools/mpremote/mpremote/transport.py

```diff
@@ -55,14 +56,10 @@ def __init__(self, status_code, error_output):
 # Takes a Transport error (containing the text of an OSError traceback) and
 # raises it as the corresponding OSError-derived exception.
 def _convert_filesystem_error(e, info):
-    if "OSError" in e.error_output and "ENOENT" in e.error_output:
-        return FileNotFoundError(info)
-    if "OSError" in e.error_output and "EISDIR" in e.error_output:
-        return IsADirectoryError(info)
-    if "OSError" in e.error_output and "EEXIST" in e.error_output:
-        return FileExistsError(info)
-    if "OSError" in e.error_output and "ENODEV" in e.error_output:
-        return FileNotFoundError(info)
```

Comment:
It looks like this one is not automatically converted, ie:
```py
OSError(errno.ENODEV, 'filename')
```
just returns:
```py
OSError(19, 'filename')
```
So maybe it needs to be handled explicitly?

Reply:
> I'm of the opinion that the conversion from `ENODEV` into `FileNotFoundError` ought to be _removed_.

Yes, that sounds reasonable to me.